posts/2021-11-09-now-with-more-elixir.md

```diff
@@ -0,0 +1,208 @@
+%{
+author: "Sean Callan",
+author_link: "https://github.com/doomspork",
+tags: ["Announcement"],
+date: ~D[2021-11-09],
+title: "Now With More Elixir!",
+excerpt: """
+How Elixir School migrated from Jekyll to a Phoenix powered site
+"""
+}
+
+---
+
+# Now With More Elixir!
+
+If you follow us on Twitter then you may have gotten a sneak peek at the new site before today's big launch but for those who haven't followed along we thought we'd cover this exciting new chapter in our history.
+
+## TL;DR The History
+
+The very first commit for Elixir School took place the day after my birthday many moons ago now: May 31, 2015. Since then the project has evolved from me to over 500 individuals who've contributed new content, translations, and other improvements. As things grew it became clear we needed to migrate from markdown in a repo, and given the lack of options in the Elixir space at the time, we selected Jekyll. Jekyll got us online and continues to serve us well provide a static site that handles 10s of thousands of weekly users all across the globe.
+
+But there's something about the place to go for learning Elixir to be built on a Ruby tool. Not to mention the simplicity of a static site is great but hampers some of the cooler ideas we've had.
+
+Fast forward six years later and there's a vibrant community around Elixir and with this wealth of experience, perspective, and background has come some really wonderful tooling.
+
+Without further ado let's look at how we got from Jekyll to Phoenix!
+
+## Jekyll to Phoenix
+
+### Nimble Publisher
+
+The single biggest hurdle in our journey from Jekyll to Phoenix has been figuring out the best way to migrate the existing content without rewriting it, losing contributor history, or adding additional burdens on contributors. Enter Dashbit's NimblePublisher library!
+
+We'll be covering our usage of NimblePublisher in follow-on blog posts.
+
+### Custom mix Tasks
+
+Custom mix tasks are employed for two main purposes today: sitemap generation and the rss feed. These two tasks leverage the content module we built with NimblePublisher.
```

Comment:
`RSS`,  in upper case, I think it's more correct :)

Reply:
```suggestion
We employ custom mix tasks for two main purposes today: generating a sitemap and RSS feed. These two tasks leverage the content modules we built with NimblePublisher.
```

Here I just switched the from passive voice to active voice - that is a subjective/preference thing though so feel free to disregard. 

I also pluralized "module" since we have lesson and post modules (a technicality/nit for sure).